src/w3c/templates/sotd.js

```diff
@@ -3,308 +3,247 @@ import html from "hyperhtml";
+      ? html`
+          <p>
+            This document is merely a W3C-internal
+            ${conf.isMO ? "member-confidential" : ""} document. It has no
+            official standing of any kind and does not represent consensus of
+            the W3C Membership.
+          </p>
+          ${opts.additionalContent}
+        `
+      : html`
+          <p><em>${[conf.l10n.status_at_publication]}</em></p>
+          ${conf.isSubmission
+            ? noteForSubmission(conf, opts)
+            : html`
+                ${!conf.sotdAfterWGinfo ? opts.additionalContent : ""}
+                ${!conf.overrideStatus
+                  ? html`
+                      <p>
+                        This document was published by ${[conf.wgHTML]} as
+                        ${conf.anOrA} ${conf.longStatus}.
+                        ${conf.notYetRec
+                          ? "This document is intended to become a W3C Recommendation."
+                          : ""}
+                      </p>
+                      ${linkToCommunity(conf, opts)}
+                      ${conf.isCR || conf.isPER || conf.isPR
                         ? html`
                             <p>
-                              This document was published by ${[conf.wgHTML]} as
-                              ${conf.anOrA} ${conf.longStatus}.
-                              ${
-                                conf.notYetRec
-                                  ? "This document is intended to become a W3C Recommendation."
-                                  : ""
-                              }
-                            </p>
-                            ${linkToCommunity(conf, opts)}
-                            ${
-                              conf.isCR || conf.isPER || conf.isPR
-                                ? html`
-                                    <p>
-                                      ${
-                                        conf.isCR
-                                          ? `
+                              ${conf.isCR
+                                ? `
                   W3C publishes a Candidate Recommendation to indicate that the document is believed to be
                   stable and to encourage implementation by the developer community. This Candidate
                   Recommendation is expected to advance to Proposed Recommendation no earlier than
                   ${conf.humanCREnd}.
                 `
-                                          : ""
-                                      }
-                                      ${
-                                        conf.isPER
-                                          ? html`
-                                              W3C Advisory Committee Members are
-                                              invited to send formal review
-                                              comments on this Proposed Edited
-                                              Recommendation to the W3C Team
```

Comment:
The error is because previously it was wrapped and now is restored🤣

Reply:
We should consider using `innerText` in this case.

Reply:
Maybe in the test we can normalize checking without the spaces? I’m fine with any solution tbh.

Reply:
Just merged in the other sotd related pull request. Might cause conflict.